lib/template-compiler.js

```diff
@@ -58,6 +58,12 @@ module.exports = function (html) {
       options.transformToRequire
     )
   }
+  var compilerVersion = require('vue-template-compiler/package.json').version
+  var vueVersion = require('vue').version
```

Comment:
probably use `vue/package.json` here too to avoid evaluating Vue itself on startup.